Resources/Textures/_CM14/Objects/Tools/crowbar.rsi/meta.json

```diff
@@ -0,0 +1,41 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "equipped-BELT Taken from tgstation at https://github.com/tgstation/tgstation/commit/eea0599511b088fdab9d43e562210cdbd51c6a98. Everything else Taken from cmss13 at https://github.com/cmss13-devs/cmss13/blob/c8c7cb927b013398cb4af9794783e964107513f5/icons/obj/items/items.dmi.",
```

Comment:
The license is fine assuming they both use the same license, however the first github link should include the file path, resembling the second link in this line which is done correctly

This goes for all the meta.json in this pr but Im not gonna note all of them to save time

I also noticed youre using your own custom inhand sprites If I saw that right, which is perfectly fine, but just so youre aware those sprites do exist and there is a .dmi file for all inhand items, split between left and right hsnd